SanDiegoCountySheriff/Cal-RIPA
# (4071) Create Base Vue 2 Application

This PR addresses incorporating an updated Vue application base code into the existing UI folder.

It includes:

1. - Vue 2
2. - ESLint (Standard) and Prettier support
3. - Vue-Router
4. - Vuex
5. - Storybook
6. - Tailwind CSS
7. - Vuetify

To test:

1. Pull down latest code
2. Manually delete node_modules and package-lock-json in the UI folder
3. Ensure you are running Node version 12.13.0 or higher
4. `cd` into the UI folder and open Terminal
4. Run `npm install --legacy-peer-deps --save` (be sure you are in the UI folder)
5. Run `npm run serve` and verify the application runs. It will be a default Vue/Vuetify screen
6. Run `npm run storybook` and verify the Storybook runs. You should see 2 stories to manage Cities and Beats.
